src/microbe_stage/Patch.cs

```diff
@@ -147,7 +147,7 @@ public float GetTotalChunkCompoundAmount(Compound compound)
 
             if (chunk.Density > 0 && chunk.Compounds.ContainsKey(compound))
             {
-                result += chunk.Density * chunk.Compounds[compound].Amount;
+                result += chunk.Density;
```

Comment:
Why did you change this? This is for showing the compound amounts in the GUI (and maybe auto-evo uses this). But the chunk size as well as the density determines how much total compounds are available, right?